src/Spectre.libClassifier.Tests/OpenCvDatasetTest.cpp

```diff
@@ -195,7 +221,7 @@ TEST_F(OpenCvDatasetTest, check_getting_labels_to_mat)
 
 TEST_F(OpenCvDatasetTest, check_if_creating_copy)
 {
-	std::unique_ptr<OpenCvDataset> result = nullptr;
+	std::unique_ptr<OpenCvDataset> result;
```

Comment:
Indents are still broken in whole file. Try to use resharper code cleanup or VS (Edit -> Untabify or something like this).